src/app/resources/iis_scripts/local_http.ps1

```diff
@@ -4,70 +4,83 @@ param(
 )
 
 $ErrorActionPreference = "Stop"
+Add-Type -AssemblyName System.Net.Http
 $contentEncoding = [System.Text.Encoding]::UTF8
 
-function stringify($content) {
-    if ((!$content) -or ($content -is [System.String])) {
+function stringify([System.Byte[]]$content) {
+    if (!$content) {
         return $content
     }
-
-    if ($content -is [System.Byte[]]) {
-        return [System.Convert]::ToBase64String($content)
-    }
-
-    return $content.ToString()
+    return [System.Convert]::ToBase64String($content).ToString()
 }
 
 ## same order as @angular/http/enums/RequestMethod
+## Note that index 0 maps to GET, so if $reqObj.method is null we would default to GET
 $requestMethods = @(
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Get,
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Post,
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Put,
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Delete,
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Options,
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Head,
-    [Microsoft.PowerShell.Commands.WebRequestMethod]::Patch
+    [System.Net.Http.HttpMethod]::Get,

```

Comment:
System.Net.Http is not available in powershell unless we load the System.Net.Http.dll manually. Please add some comment from which path we should load the dll file in debugging time.

Reply:
This is not a concern because currently admin api would install .NET Core.

P.S. If admin api is not used, this script is useless anyway so right now we can assume they are always there

Reply:
Okay, I got it.